downstream/modules/automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc

```diff
@@ -0,0 +1,52 @@
+[id="proc-configure-ansible-galaxy-cli-verify"]
+
+= Configuring Ansible-Galaxy CLI to verify collections
+
+You can configure Ansible-Galaxy CLI to verify collections. This assures that collections you download are approved by your organization and have not been changed after it was uploaded to  {HubName}.
+
+If a collection has been signed by the Galaxy server, the server provides ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection’s contents. You must opt into signature verification by link:https://docs.ansible.com/ansible/devel/reference_appendices/config.html#galaxy-gpg-keyring[configuring a keyring] for `ansible-galaxy` or providing the path with the `--keyring` option.
```

Comment:
Might be worth mentioning here that these are GPG detached signatures.  Later on we just say "signature" a lot, but never really clarify what the signatures actually are.

Reply:
Yes, I think that works for me.

Reply:
Got an "ok" from @sivel to make the changes above.